app/javascript/components/community/video-grid/index.tsx

```diff
@@ -108,22 +109,24 @@ function Video({ video }: VideoProps): JSX.Element {
     <>
       <button
         onClick={() => setIsOpen(true)}
-        className="grid shadow-sm p-16 bg-white rounded-8 text-left"
+        className="grid shadow-smZ1 p-16 bg-white rounded-8 text-left"
       >
         <img
           style={{ objectFit: 'cover', width: '100%', height: '150px' }}
           className="rounded-8 self-center mb-12"
           src={video.thumbnailUrl}
           alt="thumbnail"
         />
-        <h5 className="text-h5 mb-8">{video.title}</h5>
-        <div className="flex items-center text-left text-textColor6 font-semibold">
-          <Avatar
-            className="h-[24px] w-[24px] mr-8"
-            src={video.author && video.author.avatarUrl}
-          />
-          {video.author && video.author.name}
-        </div>
+        <h5 className="text-h5">{video.title}</h5>
+        {video.author && (
+          <div className="mt-auto pt-8 flex items-center text-left text-textColor6 font-semibold">
```

Comment:
`mt-auto` ensures the boxes are the same size with this anchored to the bottom. `pt-8` here rather than `mb-8` above ensures that if the whole row has no authors, there isn't just an extra 8px of whitespace at the bottom.